src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1,57 +1,9 @@
-UPDATE tab_rate_generic_parameters SET alias='ActiveRates_para_IN' WHERE calculation_method='Lumen_PKSim' AND formula_rate='PARAM_OralMassAbsorbed_RHS_uil' AND path_id=2146 AND parameter_name='Sum of active process rates mucosa to lumen';
-INSERT INTO tab_rate_generic_parameters(calculation_method,formula_rate,path_id,parameter_name,alias) VALUES('Lumen_PKSim','PARAM_OralMassAbsorbed_RHS_uil',2151,'Sum of active process rates lumen to mucosa','ActiveRates_trans_OUT');
-UPDATE tab_rate_generic_parameters SET alias='ActiveRates_trans_IN' WHERE calculation_method='Lumen_PKSim' AND formula_rate='PARAM_OralMassAbsorbed_RHS_uil' AND path_id=2151 AND parameter_name='Sum of active process rates mucosa to lumen';
-INSERT INTO tab_rate_generic_parameters(calculation_method,formula_rate,path_id,parameter_name,alias) VALUES('Lumen_PKSim','PARAM_OralMassAbsorbed_RHS_uje',2126,'Sum of active process rates lumen to mucosa','ActiveRates_para_OUT');
-UPDATE tab_rate_generic_parameters SET alias='ActiveRates_para_IN' WHERE calculation_method='Lumen_PKSim' AND formula_rate='PARAM_OralMassAbsorbed_RHS_uje' AND path_id=2126 AND parameter_name='Sum of active process rates mucosa to lumen';
-INSERT INTO tab_rate_generic_parameters(calculation_method,formula_rate,path_id,parameter_name,alias) VALUES('Lumen_PKSim','PARAM_OralMassAbsorbed_RHS_uje',2131,'Sum of active process rates lumen to mucosa','ActiveRates_trans_OUT');
-UPDATE tab_rate_generic_parameters SET alias='ActiveRates_trans_IN' WHERE calculation_method='Lumen_PKSim' AND formula_rate='PARAM_OralMassAbsorbed_RHS_uje' AND path_id=2131 AND parameter_name='Sum of active process rates mucosa to lumen';
+ALTER TABLE tab_process_descriptor_conditions ADD COLUMN comment;
+INSERT INTO tab_process_descriptor_conditions(process,tag,tag_type,should_have,comment) VALUES('ActiveBiDirectionalSpecificLumenMucosaIntracellular_MM','Feces','TARGET',0,NULL);
+INSERT INTO tab_process_descriptor_conditions(process,tag,tag_type,should_have,comment) VALUES('ActiveBiDirectionalSpecificLumenMucosaIntracellular_MM','Intracellular','SOURCE',1,NULL);
+INSERT INTO tab_process_descriptor_conditions(process,tag,tag_type,should_have,comment) VALUES('ActiveBiDirectionalSpecificLumenMucosaIntracellular_MM','Lumen','TARGET',1,NULL);
+INSERT INTO tab_process_descriptor_conditions(process,tag,tag_type,should_have,comment) VALUES('ActiveBiDirectionalSpecificLumenMucosaIntracellular_MM','Mucosa','SOURCE',1,NULL);
+UPDATE tab_process_descriptor_conditions SET comment='we have nbhoods between Venous/Plasma and TissueOrgan/Interstitial in the protein model (and we only want active transports between plasma and interstitial of the same organ)' WHERE process='ActiveBiDirectionalSpecificPlasmaInterstitial_MM' AND tag='VenousBlood' AND tag_type='SOURCE';

```

Comment:
@Yuri05 Ok added a commment column so that we can remember why this is like this in the future

Reply:
cool.
btw, this PR is one of the rare examples, where DB-changes are much easier to see in **PKSimDB_diff** than in **PKSimDB_dump**.

Reply:
yep... That's why we have diff and dump :) This was an amazing addition